<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddInternalLinkDlg</class>
 <widget class="QDialog" name="AddInternalLinkDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>843</width>
    <height>432</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Internal Link</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>251</width>
     <height>411</height>
    </rect>
   </property>
   <property name="title">
    <string>Available Menu Details</string>
   </property>
   <widget class="QListWidget" name="listWidgetMenu">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>231</width>
      <height>381</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>10</y>
     <width>551</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>Link Title Settings</string>
   </property>
   <widget class="QLabel" name="labelTitle">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Title :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditTitle">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>40</y>
      <width>201</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxInternalLink">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>30</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Internal Links</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonUpdateTitle">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>70</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>Update Title</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>130</y>
     <width>551</width>
     <height>241</height>
    </rect>
   </property>
   <property name="title">
    <string>Available Menu Links</string>
   </property>
   <widget class="QLabel" name="labelLinkText">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Link Text</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditLinkText">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>40</y>
      <width>211</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonInsert">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>120</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>Insert</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonRemove">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>120</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>Remove</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonUpdate">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>120</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>Update</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidgetLinks">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>20</y>
      <width>181</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButtonClose">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>390</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
